src/connectdlg.cpp

```diff
@@ -41,51 +41,63 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
     setupUi ( this );
 
     // Add help text to controls -----------------------------------------------
-    // server list
+    // directory
+    QString strDirectoryWT = "<b>" + tr ( "Directory" ) + ":</b> " +
+                             tr ( "Selects the server list to be shown. "
+                                  "You can add custom directories in Advanced Settings." );
+    QString strDirectoryAN = tr ( "Directory combo box" );
+
+    lblList->setWhatsThis ( strDirectoryWT );
+    lblList->setAccessibleName ( strDirectoryAN );
+    cbxDirectoryServer->setWhatsThis ( strDirectoryWT );
+    cbxDirectoryServer->setAccessibleName ( strDirectoryAN );
+
+    // filter
+    QString strFilterWT = "<b>" + tr ( "Filter" ) + ":</b> " +
+                          tr ( "The server list is filtered by the given text. Note that the filter is case insensitive. "
+                               "A single # character will filter for those servers with at least one client connected." );
+    QString strFilterAN = tr ( "Filter edit box" );
+    lblFilter->setWhatsThis ( strFilterWT );
+    edtFilter->setWhatsThis ( strFilterWT );
+
+    lblFilter->setAccessibleName ( strFilterAN );
+    edtFilter->setAccessibleName ( strFilterAN );
+
+    // show all mucisians
+    chbExpandAll->setWhatsThis ( "<b>" + tr ( "Show All Musicians" ) + ":</b> " +
+                                 tr ( "If you check this check box, the musicians of all servers are shown. If you "
```

Comment:
Perhaps a little long? "Check this box to show all musicians, uncheck it to collapse the list view."

Reply:
(Given it's checked by default) How about
"Uncheck to collapse the server list to show just the server details.  Check to show all musicians on each server."

Reply:
Or "...Check to show everyone on the servers.".